scipy/io/arff/tests/test_arffread.py

```diff
@@ -57,6 +57,10 @@ def test2(self):
     def test3(self):
         # Parsing trivial file with nominal attribute of 1 character.
         self._test(test6)
+        
+    def test4(self):
+        # Parsing trivial file with trailing spaces in attribute declaration.
+        self._test(test11)
```

Comment:
Note: this looked incomplete, but does work because `test11 = pjoin(data_path, 'test11.arff')` was already present further up while `data/test11.arff` wasn't present in master.